seatunnel-connectors-v2/connector-clickhouse/pom.xml

```diff
@@ -61,6 +61,12 @@
             <artifactId>clickhouse-jdbc</artifactId>
             <version>${clickhouse.version}</version>
         </dependency>
+        <dependency>
```

Comment:
Connector are without engine, can't add it in dependency.

Reply:
> Connector are without engine, can't add it in dependency.

Okay, I removed it.

Reply:
I know you want deal with array<String>, because 
```
                ArrayData arrayData = (ArrayData) field;
                BasicType<?> elementType = ((ArrayType<?, ?>) dataType).getElementType();
                return arrayData.toObjectArray(TypeConverterUtils.convert(elementType));
```
can't work right.
Maybe you can use recursion to handle this special case.

Reply:
I mean you should remove `spark-unsafe_2.11`

Reply:
Can you tell me your wechat number?

Reply:
> Can you tell me your wechat number?

541947595